common/app/Nav/Nav.jsx

```diff
@@ -51,7 +51,7 @@ const mapStateToProps = createSelector(
     return {
       panes: panes.map(({ name, type }) => {
         return {
-          content: name,
+          content: name === 'index' ? 'Editor' : name,
```

Comment:
I also noticed that Modern challenges are the only ones where the side panel is actually called "Side Panel" and the editor is called "index" (in the Nav). Side Panel looks like it can be changed to "Lesson" with no side effects, but since the editor pane is coupled to the file key, we have to make the change to the Nav UI on the component level without affecting the pane name (short of changing the file key to "Editor" in all the seed code). So this is a little inelegant and couples the name determination to this specific component, but gives us nice nav consistency across the entire app. Does this work for you?

Reply:
This is by design and there is already an issue for it. The name comes from the challenge, so creators can call the file what ever they want. 

The Side Panel can be changed in the paneMap definition itself